pos_payment_terminal/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Please, try to use OCA readme fragments https://github.com/OCA/maintainer-tools/tree/master/template/module/readme